crates/core/src/protocols/evalcheck/mod.rs

```diff
@@ -2,8 +2,14 @@
 
 //! The multivariate evalcheck polynomial protocol.
 //!
-//! Being largely intended for claims coming from sumcheck/zerocheck invocations, this module only supports
-//! multilinear composites and not general multivariate polynomials.
+//! This protocol can be used to verify a list of multilinear composite claims in the [`crate::oracle::MultilinearOracleSet`].
```

Comment:
Not composite claims, these are evaluation claims on the multilinears in the oracle set.